pkg/controller/risingwave_controller.go

```diff
@@ -27,23 +27,21 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/runtime/schema"
-	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/client-go/tools/record"
```

Comment:
Having a look at the PR... Could you please add some comments what scale view does? Just one sentence as a doctoring in the `scaleview` package  or a link to a quib doc would be good, IMHO. I know this is not part of your actual PR. I was just confused when I tried to look this up.

Reply:
Also see https://github.com/risingwavelabs/risingwave-operator/issues/231 
Unrelated to your PR. Can be done in a different PR by someone else.

Reply:
> Having a look at the PR... Could you please add some comments what scale view does? Just one sentence as a doctoring in the scaleview package or a link to a quib doc would be good, IMHO. I know this is not part of your actual PR. I was just confused when I tried to look this up.

There's an [RFC](./docs/rfc/0004_component_groups_scale_view.md) that elaborates the intention and details of the scale view. I should add a link to that in the `scaleview` package. But for quip, it's an internal doc source so we can't really reference it here.

Reply:
Okay, understand. I think it would still be helpful to add an explainer what this component does. Just a quick comment should be enough.